lib/numberpicker.dart

```diff
@@ -257,10 +269,6 @@ class NumberPicker extends StatelessWidget {
   }
 
   Widget _integerListView(ThemeData themeData) {
-    TextStyle defaultStyle = themeData.textTheme.body1;
```

Comment:
We should still use those styles if user didn't provide any styles.
I believe it go like that:
```
TextStyle defaultStyle = textStyle ?? themeData.textTheme.body1;
TextStyle selectedStyle =
       selectedTextStyle ?? themeData.textTheme.headline.copyWith(color: themeData.accentColor);
```
Same in other ocurrences

Reply:
Also, could you undo all new formatting? :) So that in one PR we can only see those changes that are important.

Reply:
sry for that format problem :\ 
I didn't change them , this ide decide to change them :(
btw I fix them

Reply:
any news on this?

Reply:
It's complete, we waiting for accepting the pull request :)